app/controllers/webauthn_verifications_controller.rb

```diff
@@ -1,13 +1,17 @@
+# This controller is for the user interface Webauthn challenge after a user follows a link generated
+# by the APIv1 WebauthnVerificationsController (controllers/api/v1/webauthn_verifications_controller).
 class WebauthnVerificationsController < ApplicationController
-  before_action :set_user
+  before_action :set_verification, :set_user
+  after_action :expire_webauthn_verification, only: :authenticate
 
   def prompt
-    redirect_to root_path, alert: t(".no_webauthn_devices") if @user.webauthn_credentials.blank?
+    redirect_to root_path, alert: t("webauthn_verifications.prompt.no_webauthn_devices") if @user.webauthn_credentials.blank?
 
     @webauthn_options = @user.webauthn_options_for_get
 
     session[:webauthn_authentication] = {
-      "challenge" => @webauthn_options.challenge
+      "challenge" => @webauthn_options.challenge,
+      "user" => @user.id
```

Comment:
Is webauthn_authentication user used? I suspect a bad merge / rebase caused [this commit](https://github.com/rubygems/rubygems.org/pull/3331/commits/9fa233edbdab62f23f7043437b575234c66b3a47) to be omitted.

Reply:
Again, there during a rebase merge.

Reply:
I'll try some more interactive rebase violence to take it out.